sys-devel/dwz/dwz-0.13.ebuild

```diff
@@ -5,20 +5,18 @@ EAPI=7
 
 inherit toolchain-funcs
 
-FPKG_HASH="1624afa75b94633e03c6e1bb952fb348"
-
 DESCRIPTION="DWARF optimization and duplicate removal tool"
-HOMEPAGE="https://sourceware.org/git/?p=dwz.git;a=summary"
-SRC_URI="https://src.fedoraproject.org/repo/pkgs/dwz/${P}.tar.bz2/${FPKG_HASH}/${P}.tar.bz2"
+HOMEPAGE="https://sourceware.org/dwz"
+SRC_URI="https://sourceware.org/ftp/dwz/releases/dwz-0.13.tar.xz"
+
+S="${WORKDIR}/${PN}"
```

Comment:
Please put ```S```after ```*DEPEND``` blocks.

Reply:
In my copy of `skel.ebuild` `S` goes between `SRC_URI` and `LICENSE`

Reply:
Oops, my bad. You are right.